go/apikeys-client.go

```diff
@@ -52,6 +61,17 @@ type OwnerTeamID = apikeys.Owner_TeamId
 // OwnerOrganisationID is an owner organisation ID.
 type OwnerOrganisationID = apikeys.Owner_OrganisationId
 
+const (
+	// GetAPIKeyRpc is the method name for Get.
+	GetAPIKeyRpc = apikeys.ApiKeysService_GetApiKey_FullMethodName
+	// CreateAPIKeyRpc is the method name for Create.
+	CreateAPIKeyRpc = apikeys.ApiKeysService_CreateApiKey_FullMethodName
+	// DeleteAPIKeyRpc is the method name for Delete.
+	DeleteAPIKeyRpc = apikeys.ApiKeysService_DeleteApiKey_FullMethodName
+	// UpdateAPIKeyRpc is the method name for Update.
+	UpdateAPIKeyRpc = apikeys.ApiKeysService_UpdateApiKey_FullMethodName
+)
+
```

Comment:
Could you please separate out the constants? Or the generated docs won't pick up the docstrings

Reply:
oh god